src/ir_Fujitsu.cpp

```diff
@@ -0,0 +1,218 @@
+// Copyright 2017 Jonny Graham
+#include "ir_Fujitsu.h"
+#include <algorithm>
+#include "IRrecv.h"
+#include "IRsend.h"
+#include "IRtimer.h"
+
+// Fujitsu A/C support added by Jonny Graham
+
+
+// Fujitsu A/C
+// Ref:
+// These values are based on averages of measurements
+#define FUJITSU_AC_HDR_MARK    3224U
+#define FUJITSU_AC_HDR_SPACE   1574U
+#define FUJITSU_AC_BIT_MARK    448U
+#define FUJITSU_AC_ONE_SPACE   1182U
+#define FUJITSU_AC_ZERO_SPACE  367U
+#define FUJITSU_AC_TRL_MARK    448U
+#define FUJITSU_AC_TRL_SPACE   8100U
+
+#if SEND_FUJITSU_AC
+// Send a Fujitsu A/C message.
+//
+// Args:
+//   data: An array of bytes containing the IR command.
+//   nbytes: Nr. of bytes of data in the array. (>=FUJITSU_AC_STATE_LENGTH)
+//   repeat: Nr. of times the message is to be repeated.
+//          (Default = FUJITSU_AC_MIN_REPEAT).
+//
+// Status: BETA / Appears to be working.
+//
+void IRsend::sendFujitsuAC(unsigned char data[], uint16_t nbytes,
+                           uint16_t repeat) {
+  // Set IR carrier frequency
+  enableIROut(38);
+  for (uint16_t r = 0; r <= repeat; ++r) {
+    // Header
+    mark(FUJITSU_AC_HDR_MARK);
+    space(FUJITSU_AC_HDR_SPACE);
+    // Data
+    for (uint16_t i = 0; i < nbytes; i++)
+    sendData(FUJITSU_AC_BIT_MARK, FUJITSU_AC_ONE_SPACE,
+         FUJITSU_AC_BIT_MARK, FUJITSU_AC_ZERO_SPACE,
```

Comment:
(Style nit-pick)
Please line up with bracket above.
e.g.
```
      sendData(FUJITSU_AC_BIT_MARK, FUJITSU_AC_ONE_SPACE,
               FUJITSU_AC_BIT_MARK, FUJITSU_AC_ZERO_SPACE,
               data[i], 8, false);
```

i.e. Line up with column after the '('.